x/observer/keeper/msg_server_update_observer.go

```diff
@@ -81,7 +81,7 @@ func (k Keeper) CheckUpdateReason(ctx sdk.Context, msg *types.MsgUpdateObserver)
 	case types.ObserverUpdateReason_AdminUpdate:
 		{
 			// Group admin is authorized to update observer
```

Comment:
```suggestion
			// Operational policy is required to update an observer for admin update 
```

Reply:
Alos I think this should be policyAdmin , this affects the observer set , and we would almost never need to do this . Observers if tombstoned have the option to select `ObserverUpdateReason_Tombstoned` reason

Reply:
Sounds good to use group admin [ba9f135](https://github.com/zeta-chain/node/pull/1935/commits/ba9f13515af948497c882c29ea761be17bacecd9) if sensitive and not frequently used